testsuite/MDAnalysisTests/analysis/test_persistencelength.py

```diff
@@ -92,6 +92,11 @@ def test_plot_with_ax(self, p_run):
         ax2 = p_run.plot(ax=ax)
 
         assert ax2 is ax
+    
+    def test_current_axes(self, p_run):
+        ax2 = p_run.plot()
+        fig, ax = plt.subplots()        
```

Comment:
Thanks for changing this -- however, because you create this after you call `.plot()`, again, the test will always pass but for the wrong reasons. You want to test that `.plot()` does not get your current axes that you have already created before calling `.plot()`.